kolibri/plugins/coach/assets/src/views/lessons/LessonsRootPage.vue

```diff
@@ -96,10 +96,14 @@
                         :style="{ display: 'inline-block', marginLeft: '6px' }"
                         :size="26"
                       />
+                      <span :id="`toggle-lesson-visibility-label-${lesson.id}`">
+                        Toggle visibility for lesson {{ lesson.name }}
+                      </span>
                       <KSwitch
-                        v-else
-                        :key="`switch-${lesson.id}`"
+                        v-else:
+                        key="`switch-${lesson.id}`"
                         name="toggle-lesson-visibility"
+                        :ariaLabelledBy="`toggle-lesson-visibility-label-${lesson.id}`"
                         label=""
```

Comment:
Since it is set as an empty string, is the `label` property still required now that we have the `ariaLabelledBy ` prop in use?